tests/test_filter_datasets.py

```diff
@@ -294,3 +341,109 @@ def test_stratified_crossval_splits_multiple_attributes(many_stimuli, crossval_f
     assert sum(len(f.x) for f in train_fixations) == (crossval_folds - val_folds - test_folds) * len(fixations.x)
 
     assert len(train_stimuli) == crossval_folds
+
+
+@pytest.mark.parametrize('attribute_name , attribute_value, invert_match', [('dva', 1, False)])
+def test_filter_stimuli_by_attribute_dva(file_stimuli_with_attributes, fixation_trains, attribute_name, attribute_value, invert_match):
+    fixations = fixation_trains[:]
```

Comment:
why is this needed?

Reply:
ah, you want to use the name `fixations`? Then I see

Reply:
no its not about the name, when you gave the skeleton of these functions, you wrote that for the function `filter_stimuli_by_attribute` the fixations argument should be of the class `Fixations`

Reply:
ah :). But `FixationTrains` is a subclass of `Fixations`, hence fixations is already a `Fixations` instance :). However I agree that sometimes it's good to make sure that it's not more than `Fixations`. Here, it shouldn't matter

Reply:
so should I change it to `FixationTrains` for this case?

Reply:
you could, but I'm fine with both.

Reply:
okay I'll leave it for `Fixations` for now.